### ISSUE TITLE
Кнопка Like

### ISSUE THREAD

**Issue by twin55** [itroom18/Tester#56]
### Пожалуйста, подробно опишите Вашу идею

Сейчас взаимодействие с кнопкой лайка очень не импактное, предлагаю сделать изначально сердечко inline а при клике середечко закрашиивается, тогда будет более понятно какой работе я поставил лайк.
![1](https://github.com/itroom18/Tester/assets/163152891/015d0005-27f9-4cf0-bf6e-178f0f6c596d)
![2](https://github.com/itroom18/Tester/assets/163152891/5681a0dc-05a4-4ae2-8abe-c6564b6f9579)

**Comment by wyrmusgod23** [comment]
Да, согалсны, сделать просто контур и при Лайке перекрашивается полностью в Желтый
